Descent3/CMakeLists.txt

```diff
@@ -309,3 +309,7 @@ endif()
 if(WIN32)
   set_target_properties(Descent3 PROPERTIES WIN32_EXECUTABLE ON)
 endif()
+
+if(BUILD_TESTING)
+  add_subdirectory(tests)
```

Comment:
If possible I'd like to have all tests in a single place (root tests/ folder). I guess you need to include the Descent directory to find your imports there

Reply:
I'm slowly working on isolating modules from each other in order to able test them as static libraries with available APIs. For that I'd like keep this structure with own tests for each component.

Reply:
no problem